nc/models.py

```diff
@@ -265,7 +265,7 @@ def census_profile(self):
 """  # noqa
 
 
-class StopSummary(pg.ReadOnlyMaterializedView):
+class StopSummary(CachingMixin, pg.ReadOnlyMaterializedView):
```

Comment:
I'd be okay leaving these off TBH, and just relying on the `cache_page` decorators